### DISCUSSION TITLE
Lucene Synonyms

### DISCUSSION THREAD

**Discussion by kevinsmgov** [OrchardCMS/OrchardCore#11473] [Q&A]
Has anyone enhanced Orchard Core's Lucene interface to enhance synonym indexing/searching? I have seen references to something called "wordnet" in the Lucene documentation but I'm not sure how I would plug that into Orchard Core.

**Comment by Skrypt** [comment]
Nothing is done yet on this part. Please share any information if anything. I'm interested.

**Reply by Skrypt** [reply]
Another one here that might be interesting for you if you go that path : https://github.com/OrchardCMS/OrchardCore/issues/3199
